src/main/java/com/targomo/client/api/response/TimeResponse.java

```diff
@@ -125,14 +148,33 @@ public TravelOptions getTravelOptions() {
 		return travelOptions;
 	}
 	
+	/**
+	 * Get travel weights from each source point to each target point.
+	 * @return map from each source to (targets, travel times)
+	 */
+	public Map<Coordinate, Map<Coordinate, TravelWeight>> getTravelWeights() {
+		return this.travelWeights;
+	}
+
 	/**
 	 * Get travel times from each source point to each target point.
 	 * @return map from each source to (targets, travel times)
 	 */
 	public Map<Coordinate, Map<Coordinate, Integer>> getTravelTimes() {
-		return this.travelTimes;
+
+		Map<Coordinate, Map<Coordinate, Integer>> filteredMap = new HashMap<>();
+
+		this.travelWeights.entrySet().forEach(entry ->
+				filteredMap.put(
+						entry.getKey(),
+						entry.getValue().entrySet().stream()
+						.collect(Collectors.toMap(Map.Entry::getKey, e -> e.getValue().getTravelTime())))
+		);
+
+		return filteredMap;
```

Comment:
can you make this a kind of a lazy getter? (so it will only be calculated once even if it is requested twice)
see https://www.projectlombok.org/features/GetterLazy